cmd/controller/main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+	"octavius/internal/controller/command"
+	"octavius/internal/controller/config"
+	"octavius/internal/pkg/log"
+)
+
+func main() {
+	logLevel := config.Config().LogLevel
+	//TODO: add log file path if any.
+	log.Init(logLevel, "")
```

Comment:
Given, log.Init has return type of error, should we not handle errors here?

Reply:
Good catch!, I will add that.

Reply:
updated.